include/cppast/compile_config.hpp

```diff
@@ -27,8 +27,14 @@ enum class cpp_standard
     cpp_2a,
     cpp_20,
     cpp_2b,
+    c_89,
```

Comment:
Nitpick: add an empty line between the C++ and C versions to separate them better.